src/main/scala/com/cognite/spark/datasource/CdpRelation.scala

```diff
@@ -42,7 +43,7 @@ abstract class CdpRelation[T <: Product: Decoder](config: RelationConfig, shortN
     with Serializable
     with PrunedFilteredScan
     with CdpConnector {
-  @transient lazy private val itemsRead =
+  @transient lazy val itemsRead: Counter =
```

Comment:
We don't need to expose this, see `getNumberOfRowsRead` in SparkTest for how to retrieve those metrics through the metric system.

Reply:
I see you're already using that, so this should still be ok as private, I think.